shared/js/dataDriver.js

```diff
@@ -343,11 +359,17 @@ loop.DataDriver = function() {
       // Determine the clock skew before continuing initialization.
       this.update("meta", "lastConnect").then(({ timestamp }) => {
```

Comment:
Or, alternately, if you think the splitting makes sense, but isn't worth doing now, I'd suggest updating the JSDoc and probably putting an XXX in the code to that effect.